notesapi/v1/models.py

```diff
@@ -0,0 +1,145 @@
+import json
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.conf import settings
+from django.db.models import signals
+from django.dispatch import receiver
+from elasticutils.contrib.django import Indexable, MappingType
+
+
+
+class Note(models.Model):
+    """
+    Annotation model.
+    """
+    # no indexes here because retrieval is taken care of by ElasticSearch
+    user_id = models.CharField(max_length=255, help_text="Anonymized user id, not course specific")
+    course_id = models.CharField(max_length=255)
+    usage_id = models.CharField(max_length=255, help_text="ID of XBlock where the text comes from")
+    quote = models.TextField(default="")
+    text = models.TextField(default="", help_text="Student's thoughts on the quote")
+    ranges = models.TextField(default="", help_text="JSON, describes position of quote in the source text")
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    def clean(self, note):
```

Comment:
my [earlier comment concerning with this API](https://github.com/edx/edx-notes-api/pull/9#discussion_r22374671) still stands, any plans to address it?